lib/action_subscriber/bunny/subscriber.rb

```diff
@@ -15,6 +15,7 @@ def auto_pop!
               :content_type => properties[:content_type],
               :delivery_tag => delivery_info.delivery_tag,
               :exchange => delivery_info.exchange,
+              :headers => properties.headers,
```

Comment:
I should probably add an integration test for both `subscribe` and `pop` modes to make sure that the headers always come through correctly and have a consistent type (ie `Hash[String => String]`)